ipc/cli/src/commands/subnet/list_validators.rs

```diff
@@ -23,23 +20,20 @@ impl CommandLineHandler for ListValidators {
     async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
         log::debug!("list validators with args: {:?}", arguments);
 
-        let url = get_ipc_agent_url(&arguments.ipc_agent_url, global)?;
-        let json_rpc_client = JsonRpcClientImpl::new(url, None);
+        let provider = get_ipc_provider(global)?;
+        let subnet = SubnetID::from_str(&arguments.subnet)?;
 
-        let params = QueryValidatorSetParams {
-            subnet: arguments.subnet.clone(),
-            epoch: None,
+        let gateway_addr = match &arguments.gateway_address {
```

Comment:
same as previous PR, I think we dont need gateway address anymore

Reply:
Tracked in #316